extensions/wikia/CuratedContent/CuratedContentController.class.php

```diff
@@ -433,6 +433,34 @@ public function getCuratedContentQuality() {
 		}
 	}
 
+	public function getData( ) {
+		global $wgWikiaCuratedContent;
+		$data = [];
+
+		if ( is_array( $wgWikiaCuratedContent ) && !empty( $wgWikiaCuratedContent ) ) {
+			foreach ( $wgWikiaCuratedContent as $section ) {
+				// sections
+				if ( !empty( $section['title'] ) && empty( $section['featured'] ) ) {
+					list( $_image_id, $url ) = CuratedContentSpecialController::findImageIfNotSet( $section['image_id'] );
```

Comment:
It would be better to use `CuratedContentHelper::getImageUrl` after merging https://github.com/Wikia/app/pull/7862 we just need to adjust width and height passed to ImageServing)